src/index.ts

```diff
@@ -194,23 +156,25 @@ async function run() {
         if (latest_block.date < finalTime) {
             console.log(`block number ${latest_block.blockNumber}: ${latest_block.transactions} transactions`);
             total_transactions += latest_block.transactions;
-            total_blocks ++;
+            total_blocks++;
         }
     }
 
     let tps = (total_transactions * 1000) / diff;
 
     console.log(`TPS from ${total_blocks} blocks: ${tps}`);
 
-    if (MEASURE_FINALISATION && !prunedFlag) {
+    if (measureFinalisation && !prunedFlag) {
         let break_condition = false;
         let attempt = 0;
         while (!break_condition) {
```

Comment:
Since this is now a function a `while(true)` plus `return`s might simplify the code.

Reply:
changed